ttnctl/cmd/applicationsPayloadFunctions.go

```diff
@@ -0,0 +1,149 @@
+// Copyright © 2016 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"golang.org/x/net/context"
+
+	"github.com/TheThingsNetwork/ttn/core"
+	"github.com/TheThingsNetwork/ttn/ttnctl/util"
+	"github.com/spf13/cobra"
+)
+
+// applicationsPayloadFunctionsCmd represents the applicationsPayloadFunctions command
+var applicationsPayloadFunctionsCmd = &cobra.Command{
+	Use:   "pf",
+	Short: "Show the payload functions",
+	Long: `ttnctl applications pf shows the payload functions for decoding,
+converting and validating binary payload.
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+		appEUI := util.GetAppEUI(ctx)
+
+		manager := getHandlerManager()
+		res, err := manager.GetPayloadFunctions(context.Background(), &core.GetPayloadFunctionsReq{
+			AppEUI: appEUI.Bytes(),
+		})
+		if err != nil {
+			ctx.WithError(err).Fatal("Could not get payload functions")
+		}
+
+		ctx.Info("Has decoder function")
+		fmt.Println(res.Decoder)
+
+		if res.Converter != "" {
+			ctx.Info("Has converter function")
+			fmt.Println(res.Converter)
+		}
+
+		if res.Validator != "" {
+			ctx.Info("Has validator function")
+			fmt.Println(res.Validator)
+		} else {
+			ctx.Warn("Does not have validator function")
+		}
+	},
+}
+
+// applicationsSetPayloadFunctionsCmd represents the applicationsSetPayloadFunctions command
+var applicationsSetPayloadFunctionsCmd = &cobra.Command{
+	Use:   "set [decoder.js]",
+	Short: "Set payload functions",
+	Long: `ttnctl applications pf set sets the decoder, converter and validator
+function by loading the specified files containing JavaScript code.
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+		appEUI := util.GetAppEUI(ctx)
+
+		if len(args) == 0 {
+			cmd.Help()
+			return
+		}
+
```

Comment:
The decoder function is mandatory, both the converter and the validator function are optional so they are flags. Therefore, bullet 2-4 doesn't work.